src/parameterizations/vertical/MOM_ALE_sponge.F90

```diff
@@ -847,11 +847,11 @@ subroutine apply_ALE_sponge(h, dt, G, GV, US, CS, Time)
           ! Build the source grid
           zTopOfCell = 0. ; zBottomOfCell = 0. ; nPoints = 0; hsrc(:) = 0.0; tmpT1d(:) = -99.9
           do k=1,nz_data
-             if (mask_z(CS%col_i(c),CS%col_j(c),k) == 1.0) then
```

Comment:
What does this change have to do with ice shelves? It's not inside a `frac_shelf` block so seems unrelated to the mods for tc4.

Reply:
Nothing to do with ice shelves, but triggered in tc4 when the sponges are turned on.